### PR TITLE
design refine - sidebar, layout

### PR DESCRIPTION
**Pull request by brendansudol**
this PR continues the design refinement work and focuses primarily on the sidebar component

please merge https://github.com/18F/cms-hitech-apd/pull/710 first

preview:
design-refine

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author

### This feature is done when...
- [x] Design has approved the experience

### CODE REVIEW

**Comment by brendansudol** [comment]
the code changes here are quite noisy primarily because i had to regenerate all the state icons with a new color

File: web/src/containers/TopBtns.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    const { authenticated } = this.props;
+
+    return (
+      <header className="clearfix mb2">
+        <div className="right h5">
+          {authenticated ? (
+            <Fragment>
+              <button type="button" className="btn btn-primary px1 py-tiny mr1">
+                {t('dashboard')}
+              </button>
```
Comment (mgwalker) [inline web/src/containers/TopBtns.js]: The "state dashboard" button is a placeholder I guess?

**Comment by mgwalker** [comment]
Holding for #710

**Comment by mgwalker** [comment]
Ready as soon as conflicts are resolved!  :tada:

File: web/src/containers/TopBtns.js (hunk 1 above)
Comment (brendansudol) [reply web/src/containers/TopBtns.js]: yes!

**Pull request merged**